src/NuGet.Core/NuGet.Packaging/Core/FrameworkNameValidatorUtility.cs

```diff
@@ -34,6 +34,60 @@ internal static bool IsValidFrameworkName(string path)
             return fx != null && fx.Framework != NuGetFramework.UnsupportedFramework.Framework;
         }
 
+        internal static bool IsDottedFrameworkVersion(string path)
+        {
+            foreach (string knownFolder in PackagingConstants.Folders.Known)
+            {
+                string folderPrefix = knownFolder + System.IO.Path.DirectorySeparatorChar;
+                if (knownFolder == "contentFiles")
+                {
+                    folderPrefix += "any" + System.IO.Path.DirectorySeparatorChar;
```

Comment:
`any` is not the only option here. 
`cs`, `fs` & `vb` as language options work too. 
I'd avoid hardcoding it if at all possible. 

The impact is just completeness, so nothing major.